filebeat/harvester/reader/json_config.go

```diff
@@ -1,10 +1,11 @@
 package reader
 
 type JSONConfig struct {
-	MessageKey    string `config:"message_key"`
-	KeysUnderRoot bool   `config:"keys_under_root"`
-	OverwriteKeys bool   `config:"overwrite_keys"`
-	AddErrorKey   bool   `config:"add_error_key"`
+	MessageKey     string `config:"message_key"`
+	KeysUnderRoot  bool   `config:"keys_under_root"`
+	OverwriteKeys  bool   `config:"overwrite_keys"`
+	AddErrorKey    bool   `config:"add_error_key"`
+	LogParseErrors bool   `config:"log_parse_errors"`
```

Comment:
Default bool value is `false`. So this would be enable by default which would be a change in behaviour I think.